src/commands/basemaps-github/create-pr.ts

```diff
@@ -29,14 +29,15 @@ function assertValidBucket(bucket: string, validBuckets: Set<string>): void {
 
 async function parseRasterTargetInfo(
   target: string,
+  elevation: boolean,
   individual: boolean,
 ): Promise<{ name: string; title: string; epsg: EpsgCode; region: string | undefined }> {
   logger.info({ target }, 'CreatePR: Get the layer information from target');
   const url = new URL(target);
   const bucket = url.hostname;
   const splits = url.pathname.split('/');
-  const epsg = Epsg.tryGet(Number(splits[1]));
-  const name = splits[2];
+  const epsg = elevation ? Epsg.tryGet(Number(splits[2])) : Epsg.tryGet(Number(splits[1]));
+  const name = elevation ? splits[3] : splits[2];
```

Comment:
parsing the path names are known to be super brittle, could we grab this information from the `collection.json` ?

Reply:
Sounds better option, we can validate collection.json first before validate these espg and name

Reply:
yeah just grab the collection.json then start looking for this information.

Reply:
I didn't seen any name from `collection.json/covering.json/source.json`. Only saw the `tilematrix` in the item.json that we could probably use to parse to epsg.

Reply:
@blacha I don't think we can get the name from Stac file atm as we only store title in it.